### ISSUE TITLE
Push MVP branch BEFORE starting stretch goals

### ISSUE THREAD

**Issue by taylorseager** [taylorseager/INDIVIDUAL-ASSIGNMENT-Team-Roster#5]
# User Story
Once MVP is met, make an MVP branch and push it to Github for the instructors to see MVP easily. Once this is complete stretch goals can be worked on. This ticket should be used to ensure MVP is met.

# Acceptance Criteria

### Authentication
- [x] As a user, who is logged out, I should only be able to see the authentication screen
- [x] As a user who is logged out, I would like to be able to click on an authentication button and login via google.
- [x] As a user who is logged in, I should not be able to see the authentication button.
- [x] As a user who is logged in, I should be able to see the Team view.
- [x] As a user who is logged in, I should be able to see a log out button

### Routing
- [x] As an authenticated user, if I click the TEAM link in the navbar, I should navigate to '/team' and see an h1 tag that says 'Team'.
- [x] As an authenticated user, if I click the NEW link in the navbar, I should navigate to '/new' and see an h1 tag that says 'Add a Member'.

### CREATE Members
- [x] As an authenticated user, when I create a member, the member object should include my uid.
- [x] As an authenticated user, I should be able to click the NEW link in the navbar that displays a form to add a new member.
- [x] As an authenticated user, when I fill out the form and submit a new member should be created in firebase and should now show in my Team view.

### READ Members
- [x] As an authenticated user, I should be able to see the Team view with all the members I have created.
- [x] As an authenticated user I should not be able to see members that were created by another user.

### UPDATE Members
- [x] As an authenticated user, I should be able to see an edit button on each member card.
- [x] As an authenticated user, when I click the edit button I should see a form with the member information pre-populated in the form.
- [x] As an authenticated user, I should be able to edit the information in the form and hit the submit button.
- [x] As an authenticated user, when I submit the edit form firebase should be updated (PATCH request) and the Team view should update

### DELETE Members
- [x] As an authenticated user, I should see a delete button on each member.
- [x] As an authenticated user, when I click a delete button that member should be removed from firebase and the Team view should update.

# Dependecies
All CRUD issue tickets should be completed
